docs/en/01-how-to-build/build_from_source.md

```diff
@@ -3,7 +3,7 @@
 ## Download
 
 ```shell
-git clone -b master git@github.com:open-mmlab/mmdeploy.git --recursive
+git clone -b 1.x git@github.com:open-mmlab/mmdeploy.git --recursive
```

Comment:
I think `dev-1.x` is better

Reply:
@lvhan028 https://github.com/open-mmlab/mmdeploy/blame/a62ed41f4457502fa152fe20d7b2e6de19ea92ec/docs/zh_cn/01-how-to-build/build_from_source.md?plain=1#L30

Reply:
I see